lib/autocop/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Autocop
-  VERSION = '0.4.3'
+  VERSION = '0.4.4'
```

Comment:
Don't add the version bump in pull requests. This is done from master when making a new gem version release